### PR TITLE
Setup Travis CI

### PR DESCRIPTION
**Pull request by eugenepeh**
Reference: https://github.com/eugenepeh/RepoSense/pull/2

### CODE REVIEW

**Comment by eugenepeh** [comment]
@harryggg Just like check

<pre>
analyzer.AnnotatorAnalyzerTest STANDARD_OUT
    Directory does not exist.
    Directory does not exist.
    cloning harryggg/GitTest...
    cloning done!
</pre>

Got all these messages from running tests on gradle. What directory is missing?

**Comment by damithc** [comment]
> Got all these messages from running tests on gradle. What directory is missing?

Most likely just an info message.  If that is the case, we should rephrase it to avoid appearing like an error message.

**Comment by harryggg** [comment]
The message is shown when there the repository is not found, which means RepoSense needs to clone it first. I will rephrase it now.

**Comment by harryggg** [comment]
LGTM.
